xrpl-wasm-stdlib/src/core/ledger_objects/traits.rs

```diff
@@ -529,5 +529,1521 @@ pub trait AccountFields: LedgerObjectCommonFields {
 
 #[cfg(test)]
 mod tests {
-    // NOTE: Will be replaced fully by the next PR that adds new unit tests.
+    use super::*;
+    use crate::core::ledger_objects::account_root::AccountRoot;
+    use crate::host::error_codes::{FIELD_NOT_FOUND, INTERNAL_ERROR, INVALID_FIELD};
+    use crate::host::host_bindings_trait::MockHostBindings;
+    use mockall::predicate::{always, eq};
+
+    mod ledger_object_common_fields {
+        use super::*;
+        use crate::host::setup_mock;
+
+        #[test]
+        fn test_get_flags_returns_u32() {
+            let mut mock = MockHostBindings::new();
```

Comment:
One will notice that this PR has a reasonably large number of lines-of-code changed. The primary culprit here is the the number of tests that adhere to approximately the same pattern seen in this test (combined with a large number of unit tests required to test each trait function). 

One way to reduce this repetition is to improve the test harness (see the [MockGuard](https://github.com/ripple/xrpl-wasm-stdlib/pull/117/changes#diff-b88a359bc721e7e59abef31a78cf0102d80f5b8ee928af0f6d576b716b9ba689R9) and it's associated machinery) to reduce boilerplate. This comes as the cost of readability/understanding, but I'm open to suggestions or preferences.